taxi/urls.py

```diff
@@ -46,6 +50,23 @@
     path(
         "drivers/<int:pk>/", DriverDetailView.as_view(), name="driver-detail"
     ),
+    path("drivers/create/", DriverCreateView.as_view(), name="driver-create"),
+    path(
+        "drivers/<int:pk>/update/",
+        DriverUpdateView.as_view(),
+        name="driver-update"
+    ),
+    path(
+        "drivers/<int:pk>/delete/",
+        DriverDeleteView.as_view(),
+        name="driver-delete"
+    ),
+    path(
+        "assign-me/<int:pk>/<int:driver_id>/",
```

Comment:
```suggestion
        "assign-me/<int:pk>/",
```

this is redundant, cause you can get user from the request